pages/osx/fsck.md

```diff
@@ -0,0 +1,20 @@
+# fsck
+
+> Check the integrity of a filesystem or repair it. The filesystem should be unmounted at the time the command is run.
+> So fsck is now essentially a wrapper that calls fsck_hfs, fsck_apfs, fsck_msdos, fsck_exfat, and fsck_udf as needed.
```

Comment:
This feels awkwardly worded. Perhaps `fsck is a wrapper that calls`... instead?

Reply:
I had the same feeling. Agree with @sbrl's suggestion.

Reply:
Feel free to do it, I deleted the repo thinking everything was done, sorry!

Reply:
Uh oh. Since you deleted the fork, we will have to close this PR, send a new PR with the changes.